x-pack/plugins/apm/public/components/app/RumDashboard/RumHome.tsx

```diff
@@ -26,6 +27,9 @@ export function RumHome() {
                 <h1>{UX_LABEL}</h1>
               </EuiTitle>
             </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <UserPercentile />
+            </EuiFlexItem>
```

Comment:
I would suggest aligning the percentile selector by the date picker. This is a global selector like the time range, so it feels more natural to place adjacent to that option.

![94430494-4dfac600-0194-11eb-9925-54daf9c1f55a](https://user-images.githubusercontent.com/4104278/94472504-24a95c80-01cb-11eb-9edf-96776b26644f.png)

cc @drewpost @katrin-freihofner

Reply:
We could choose to add a prepended label in front of the selector which displays "Aggregation" or "Agg." shortened.